src/plone/restapi/types/utils.py

```diff
@@ -76,13 +76,20 @@ def get_ordered_fields(fti):
     return (fields, ordered_fieldsets_fields)
 
 
+def get_hidden_fields_from_schema(schema):
+    hidden_fields = {}
+    for (iface, title, display_mode) in schema.getTaggedValue(MODES_KEY):
+        hidden_fields[title] = display_mode
+    return hidden_fields
+
+
 def get_fields_from_schema(schema, context, request, prefix='',
                            excluded_fields=None):
     """Get jsonschema from zope schema."""
     fields_info = OrderedDict()
     if excluded_fields is None:
         excluded_fields = []
-
+    hidden_fields = get_hidden_fields_from_schema(schema)
```

Comment:
Here I look up the hidden fields from schema.getTaggedValue(). We might want to refactor this to be more generic and catch all possible plone.autoform MODE_KEYS.